### PR TITLE
Add release packages for coq-hott

### PR DESCRIPTION
**Pull request by JasonGross**
Closes HoTT/HoTT#1132

Question: Should we delete the per-version dev packages, or leave them?

### CODE REVIEW

**Comment by palmskog** [comment]
I see no obvious reason to delete the `extra-dev` packages, if they still work.

File: released/packages/coq-hott/coq-hott.8.11/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD-2-Clause"
+build: [
+  ["bash" "-c" "./autogen.sh -skip-submodules || :"]
```
Comment (palmskog) [inline released/packages/coq-hott/coq-hott.8.11/opam]: Is the `-skip-submodules` argument really used when the code is delivered by tarball? I thought that had to do with Git submodules?

Comment (JasonGross) [reply released/packages/coq-hott/coq-hott.8.11/opam]: Ah, indeed, we don't need it.  But I don't see any reason to drop it, as it still works fine even when the code is delivered by tarball, and if we ever switch back to git, it is important to have.  I can drop it if you think it's important to do so, though.

Comment (palmskog) [reply released/packages/coq-hott/coq-hott.8.11/opam]: It's not important as long as it works fine with tarball delivery.

**Pull request merged**